author.yaml

```
@@ -40,3 +40,6 @@
 - github: Lee-juNu
   name: リリ
   bio: 韓国から来た新人エンジニアです。プログラムは広く浅く学んできたので早めに基礎をしっかり整えて立派なエンジニアになりたいです。趣味はVRゲームに関連する3DモデリングやUnityを通じたプログラミングです。
+- github: moxi1021
+  name: もいもい
+  bio: 社会人15年生、入社1年目のエンジニアです。今まではC、C++が主のミドルウェア系のS/Wを開発していました。その対向となる画面系はC#。最近は何か自分に刺さるゲームが作りたい！と思いつつ日々を過ごしています。
```

あまり S/W っていらない気もしたのですがどうでしょう

```suggestion
  bio: 社会人15年生、入社1年目のエンジニアです。今まではC、C++が主のミドルウェアを開発していました。GUIはC#が得意です。最近は何か自分に刺さるゲームが作りたい！と思いつつ日々を過ごしています。
```

---

確認ありがとうございます。
ミドルウェアだけで伝わりますね。S/Wは削除します。

---

頂いた指摘を反映しました。
ご確認よろしくお願いします。